src/current/v24.3/ldap-authentication.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Configure LDAP Authentication
+summary: Learn how to configure CockroachDB to use LDAP for user authentication with directory services like Active Directory.
+toc: true
+---
+
+CockroachDB supports authentication and authorization using LDAP (Lightweight Directory Access Protocol) with services such as Active Directory. This allows you to integrate your existing directory service with CockroachDB for centralized user management and access control.
+
+This page describes how to configure CockroachDB user authentication using LDAP. You can also configure user authorization (role-based access control) with LDAP, to set users' CockroachDB roles based on group memberships in the directory, but this is not required.
+
+## Overview
+
+LDAP authentication in CockroachDB uses a 'search and bind' approach:
+
+1. CockroachDB first connects to the LDAP server using a service account (bind DN).
+2. It searches for the authenticating user using configurable search criteria.
+3. If found, it attempts to bind to LDAP using the user's credentials.
+4. Upon successful binding, the user is authenticated to CockroachDB.
+
+This method supports both standard LDAP servers and Active Directory, with secure LDAPS connections over TLS.
```

Comment:
```suggestion
LDAP authentication be used with both standard LDAP servers and services such as Azure Active Directory. Secure LDAPS connectivity over TLS is required.
```
But maybe it belongs in Prerequisites, which is nearby already.

Reply:
```suggestion
LDAP authentication in CockroachDB works with any LDAP-compatible directory service, including Microsoft Active Directory and OpenLDAP. Secure LDAPS connectivity over TLS is supported, and recommended for production environments.
```